pkg/util/leaktest/leaktest.go

```diff
@@ -55,6 +55,9 @@ func interestingGoroutines() map[int64]string {
 		if stack == "" ||
 			strings.Contains(stack, "github.com/cockroachdb/cockroach/pkg/util/log.init") ||
 			strings.Contains(stack, "github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger") ||
+			// Ignore HTTP keep alives
+			strings.Contains(stack, "internal/poll.runtime_pollWait") ||
```

Comment:
hm, so I think `runtime_pollWait` could just be any waiting network conn -- is is possible to && this one with something that looks for `persistConn` further down to assert the it really is the http keep alive one that is sitting in the pollWait?